blueflood-core/src/test/java/com/rackspacecloud/blueflood/types/MaxValueTest.java

```diff
@@ -32,97 +39,375 @@ public void setUp() {
     }
 
     @Test
-    public void testMaxValueForDoubleMetrics() throws IOException {
-        for (double val : TestData.DOUBLE_SRC) {
-            max.handleFullResMetric(val);
-        }
-        Assert.assertTrue(max.isFloatingPoint());
-        Assert.assertEquals(90.48232472545334, max.toDouble(), 0);
+    public void fullResInitialDoubleSetsValue() {
+
+        // when
+        max.handleFullResMetric(123.45d);
```

Comment:
@shintasmith The `max` field is initialized in the `setUp()` method. Initializing it in the separate test methods instead would add a line for each test. Using a local variable for each test instead of a field would save one line total. Functionally, the tests would work exactly the same; there would be no difference. The two approaches are equivalent, and more a matter of taste. I can change it if you prefer.

The `other` variable, on the other hand, is only used in two test methods. It wouldn't really gain anything to have it as a field. Of course, that is possible to do, and would be equivalent, as above. Again, it's a matter of taste. I can change it if you think that would look better or be more readable.